go/vt/vtexplain/vtexplain.go

```diff
@@ -182,6 +199,8 @@ func Run(sql string) ([]*Explain, error) {
 		}
 
 		if sql != "" {
+			batchTime = sync2.NewBatcher(time.Duration(10 * 1000 * 1000))
```

Comment:
`batchTime` is not used and not declared before?

I didn't look at the rest of the code, but it's not clear to me yet how this batcher will be used here :)

Reply:
This probably warrants a comment, but it's resetting the batch counter declared in `vtexplain_vttablet.go` so that each query starts from batch 1.

Reply:
I added a comment here.